### ISSUE TITLE
[MNG-3987] No attempts to connect to remote repositories under Sun JDK 1.6.0.06 (i386 and x86_64) 

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#5990]
**[Andrew Lee Rubinger](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=alr@alrubinger.com)** opened **[MNG-3987](https://issues.apache.org/jira/browse/MNG-3987?redirect=false)** and commented

While using Sun jdk1.6.0_06-x86_64 as JAVA_HOME, remote dependencies are not downloaded.  Wireshark network analysis shows no TCP requests sent out of port 80.  Using jdk1.6.0_11-x86_64 resolves the issue.

Steps to duplicate (on the reporter's local environment)

1) Clean local M2 repo (ie. ~/.m2/repository)
2) Set JAVA_HOME to jdk1.6.0_06-x86_64
3) Attempt "mvn install"..Observe dependency resolution problems as artifacts and POMs cannot be downloaded.  Maven reports as not found.  The URLs noted in the trace are accessible via wget or browser.
4) Set JAVA_HOME to jdk1.6.0_11-x86_64
5) Attempt "mvn install" Dependencies will be downloaded.

---

**Attachments:**
- [maven_dependency_resolution_fail-jdk1.6.0_06-x86_64.txt](https://issues.apache.org/jira/secure/attachment/12712819/maven_dependency_resolution_fail-jdk1.6.0_06-x86_64.txt) (_5.96 kB_)
- [maven_dependency_resolution_works-jdk1.6.0_11-x86_64.txt](https://issues.apache.org/jira/secure/attachment/12712863/maven_dependency_resolution_works-jdk1.6.0_11-x86_64.txt) (_45.29 kB_)

**Comment by jira-importer** [comment]
**[Andrew Lee Rubinger](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=alr@alrubinger.com)** commented

Attached "mvn install -X" resolution failing under jdk1.6.0_06-x86_64

**Comment by jira-importer** [comment]
**[Andrew Lee Rubinger](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=alr@alrubinger.com)** commented

Attached output of "mvn install -X" using jdk1.6.0_11-x86_64

**Comment by jira-importer** [comment]
**[Benjamin Bentmann](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=bentmann)** commented

Could be interesting to find out what exception causes this debug output:

```
Downloading: http://repo1.maven.org/maven2/org/jboss/jboss-parent/3/jboss-parent-3.pom
[DEBUG] Unable to get resource 'org.jboss:jboss-parent:pom:3' from repository central (http://repo1.maven.org/maven2)
```

The just released [Maven 3.0-alpha-1](http://repo1.maven.org/maven2/org/apache/maven/maven-distribution/3.0-alpha-1/) should provide more debug information in the failing case here.
